### ISSUE TITLE
AS3 Support for Clone/Server Pools on Virtual Server

### ISSUE THREAD

**Issue by jmcalalang** [F5Networks/f5-appsvcs-extension#17]
##### ISSUE TYPE

 - Feature Idea

##### AS3 BUILD/ VERSION

```
 Future
```

##### BIGIP VERSION

```
12.1+
```

##### SUMMARY

```
Customer is looking to utilize the clone pool features of the virtual server. this would allow them to specify a declaration which contains sending traffic to other tools for IDS/IPS/ protocol inspections 
```

##### STEPS TO REPRODUCE

```
I think this would be a schema update to support these Virtual Properties:

    "clonePools": [
        {
            "name": "client_clone_pool",
            "partition": "Common",
            "context": "clientside",
            "nameReference": {
                "link": "https://localhost/mgmt/tm/ltm/pool/~Common~client_clone_pool?ver=13.1.0.8"
            }
        },
        {
            "name": "server_clone_pool",
            "partition": "Common",
            "context": "serverside",
            "nameReference": {
                "link": "https://localhost/mgmt/tm/ltm/pool/~Common~server_clone_pool?ver=13.1.0.8"
            }
        }
```
```
Full GET from VS with clone:

{
    "kind": "tm:ltm:virtual:virtualstate",
    "name": "clone_vs",
    "fullPath": "clone_vs",
    "generation": 20953,
    "selfLink": "https://localhost/mgmt/tm/ltm/virtual/clone_vs?ver=13.1.0.8",
    "addressStatus": "yes",
    "autoLasthop": "default",
    "cmpEnabled": "yes",
    "connectionLimit": 0,
    "destination": "/Common/42.42.42.42:42",
    "enabled": true,
    "gtmScore": 0,
    "ipProtocol": "tcp",
    "mask": "255.255.255.255",
    "mirror": "disabled",
    "mobileAppTunnel": "disabled",
    "nat64": "disabled",
    "rateLimit": "disabled",
    "rateLimitDstMask": 0,
    "rateLimitMode": "object",
    "rateLimitSrcMask": 0,
    "securityNatPolicy": {
        "useDevicePolicy": "no",
        "useRouteDomainPolicy": "no"
    },
    "serviceDownImmediateAction": "none",
    "source": "0.0.0.0/0",
    "sourceAddressTranslation": {
        "type": "none"
    },
    "sourcePort": "preserve",
    "synCookieStatus": "not-activated",
    "throughputCapacity": 0,
    "translateAddress": "enabled",
    "translatePort": "enabled",
    "vlansDisabled": true,
    "vsIndex": 22,
    "clonePools": [
        {
            "name": "client_clone_pool",
            "partition": "Common",
            "context": "clientside",
            "nameReference": {
                "link": "https://localhost/mgmt/tm/ltm/pool/~Common~client_clone_pool?ver=13.1.0.8"
            }
        },
        {
            "name": "server_clone_pool",
            "partition": "Common",
            "context": "serverside",
            "nameReference": {
                "link": "https://localhost/mgmt/tm/ltm/pool/~Common~server_clone_pool?ver=13.1.0.8"
            }
        }
    ],
    "policiesReference": {
        "link": "https://localhost/mgmt/tm/ltm/virtual/~Common~clone_vs/policies?ver=13.1.0.8",
        "isSubcollection": true
    },
    "profilesReference": {
        "link": "https://localhost/mgmt/tm/ltm/virtual/~Common~clone_vs/profiles?ver=13.1.0.8",
        "isSubcollection": true
    }
}

```

##### ACTUAL RESULTS

Feature Request to NA

```
Feature Request to NA
```

**Comment by kaustriaf5** [comment]
This is being tracked internally as 837.

**Comment by dstokesf5** [comment]
Development on this feature has been completed, and it should be available in the next release (3.9.0).

**Comment by dstokesf5** [comment]
3.9.0 was released a while ago. Closing.
